pkg/platform/kube/functionres/lazy.go

```diff
@@ -174,28 +183,18 @@ func (lc *lazyClient) CreateOrUpdate(ctx context.Context, function *nuclioio.Nuc
 
 	resources := lazyResources{}
 
+	// enrich function config
 	platformConfig := lc.platformConfigurationProvider.GetPlatformConfiguration()
-	for _, augmentedConfig := range platformConfig.FunctionAugmentedConfigs {
```

Comment:
I moved it to a function + added the HTTP Trigger augmented configuration specifics
to allow the function to be unit-tested